### PR TITLE
Add openapi generator

### PR DESCRIPTION
**Pull request by elias-knodel**
## Description

Generates SDKs and pushes them to this repo: https://github.com/lazybytez/jojo-bot-sdk

### CODE REVIEW

File: .github/workflows/deploy_sdk.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          -v $(pwd):/local openapitools/openapi-generator-cli generate \
+          -i docs/swagger.json \
+          -g go \
+          -o dist/go-client \
+          --package-name=jojo-discord-bot-api-client
+
+      - name: Push code into external sdk repository
+        run: |
+          git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
+          git config --global user.email "lazybytez-cloud@users.noreply.github.com"
```
Comment (pascal-zarrad) [inline .github/workflows/deploy_sdk.yml:52]: Please add the id of the noreply address. Newer GitHub accounts always also habe an ID infront.

**Review by pascal-zarrad — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_sdk.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      IS_CI: true
```
[the remark this answers is not in the document]
Comment (elias-knodel) [reply .github/workflows/deploy_sdk.yml:14]:
```suggestion

```

File: .github/workflows/deploy_sdk.yml (hunk 1 above)
Comment (elias-knodel) [reply .github/workflows/deploy_sdk.yml:51]: Yeah, what is the ID?

Comment (elias-knodel) [reply .github/workflows/deploy_sdk.yml:51]:
```suggestion
          git config --global user.email "83846103+lazybytez-cloud@users.noreply.github.com"
```

**Pull request merged**